src/proxy_configuration.js

```diff
@@ -196,6 +196,10 @@ export class ProxyConfiguration {
         this.log = defaultLog.child({ prefix: 'ProxyConfiguration' });
         this.config = config;
         this.isManInTheMiddle = false;
+
+        if (proxyUrls && proxyUrls.some((url) => url.includes('apify.com'))) {
+            this.log.warning('Some Apify proxy features may work incorrectly. Please consider setting up Apify properties instead of `proxyUrls`.');
```

Comment:
by "Apify properties" you mean env vars? maybe we should suggest login via CLI?

Reply:
Anything. Can be env vars, cli, `groups` and `password` etc.

Reply:
maybe we could add a link to the docs to make it more clear

https://sdk.apify.com/docs/guides/proxy-management#apify-proxy-configuration

or maybe better

https://bit.ly/3l99Ijj

Reply:
```suggestion
            this.log.warning('Some Apify proxy features may work incorrectly. See https://sdk.apify.com/docs/guides/proxy-management#apify-proxy-configuration');
```

Reply:
?

Reply:
i'd keep the other sentence too